README.md

```diff
@@ -142,16 +155,13 @@ The following table shows all beans with their respective name (for the `example
 | Bean Name                              | Bean Type                       | Configures by default      |
 |----------------------------------------|---------------------------------|----------------------------|
 | `accessToken` (no client prefix!)      | `AccessTokens`                  | OAuth settings             |
+| `exampleHttpMessageConverters`         | `HttpMessageConverters`         | Text and JSON              |
 | `exampleHttpClient`                    | `HttpClient`                    | Interceptors               |
 | `exampleClientHttpRequestFactory`      | `ClientHttpRequestFactory`      | Timeouts                   |
-| `exampleHttpMessageConverters`         | `HttpMessageConverters`         | Text and JSON              |
-| `exampleRestTemplate`                  | `RestTemplate`                  | Base URL and error handler |
-| `exampleRestTemplate`                  | `StupsOAuth2RestTemplate`       | Base URL and error handler |
-| `exampleRest`                          | `Rest`                          |                            |
+| `exampleRestTemplate`                  | `RestTemplate`                  | Base URL                   |
```

Comment:
I started of with removing `AsyncRestTemplate` *and* `RestTemplate` but backed off, because there are some libraries that are expecting you to work with a `RestTemplate` (e.g. [stups-spring-oauth-support-server](https://github.com/zalando-stups/stups-spring-oauth2-support/tree/master/stups-spring-oauth2-server)). That's why I've kept it in case users (aka me) need something synchronous.